### PR TITLE
#112: changed loading text to Loading GeOrchestra from Loading MapStore

### PR DESCRIPTION
**Pull request by mbarto**

### CODE REVIEW

File: indexTemplate.html [hunk 1]
```diff
@@ -115,7 +133,7 @@
       <iframe id="georchestra-header" src="/header/?active=mapstore" scrolling="no" frameBorder="0"></iframe>
       <div id="container">
           <div class="_ms2_init_spinner _ms2_init_center"><div></div></div>
-          <div class="_ms2_init_text _ms2_init_center">Loading MapStore</div>
+          <div class="_ms2_init_text _ms2_init_center">Loading GeOrchestra</div>
```
Comment (tdipisa) [inline indexTemplate.html]: Please use "geOrchestra"

File: index.html [hunk 2]
```diff
@@ -116,7 +135,7 @@
         <iframe id="georchestra-header" src="/header/?active=mapstore" scrolling="no" frameBorder="0"></iframe>
         <div id="container">
             <div class="_ms2_init_spinner _ms2_init_center"><div></div></div>
-            <div class="_ms2_init_text _ms2_init_center">Loading MapStore</div>
+            <div class="_ms2_init_text _ms2_init_center">Loading GeOrchestra</div>
```
Comment (tdipisa) [inline index.html]: Please use "geOrchestra"

File: index.html [hunk 3]
```diff
@@ -20,6 +20,23 @@
             display: flex;
             align-items: center;
         }
+        ._ms2_init_center::before {
+            content: 'Loading GeOrchestra';
```
Comment (tdipisa) [inline index.html]: Please use "geOrchestra"

File: indexTemplate.html [hunk 4]
```diff
@@ -20,6 +20,23 @@
         display: flex;
         align-items: center;
       }
+      ._ms2_init_center::before {
+          content: 'Loading GeOrchestra';
```
Comment (tdipisa) [inline indexTemplate.html]: Please use "geOrchestra"

File: index.html (hunk 2 above)
Comment (mbarto) [inline index.html]:
```suggestion
            <div class="_ms2_init_text _ms2_init_center">Loading geOrchestra</div>
```

File: index.html (hunk 3 above)
Comment (mbarto) [inline index.html]:
```suggestion
            content: 'Loading geOrchestra';
```

File: indexTemplate.html (hunk 4 above)
Comment (mbarto) [inline indexTemplate.html]:
```suggestion
          content: 'Loading geOrchestra';
```

File: indexTemplate.html (hunk 1 above)
Comment (mbarto) [inline indexTemplate.html]:
```suggestion
          <div class="_ms2_init_text _ms2_init_center">Loading geOrchestra</div>
```

**Pull request merged**